src/NUnitTestAdapter/TestLogger.cs

```diff
@@ -60,7 +60,7 @@ public void LoadingAssemblyFailedWarning(string dependentAssembly, string source
 
         public void NUnitLoadError(string sourceAssembly)
         {
-            SendErrorMessage("NUnit failed to load " + sourceAssembly);
+            SendInformationalMessage("No NUnit 2 tests found in " + sourceAssembly);
```

Comment:
See my comment on the 3.0 issue. Seems like a method named NUnitLoadError should issue error messages... either we should not use it or it should have another name.